modules/alma/lib/AlmaClient/AlmaClient.class.php

```diff
@@ -438,7 +446,7 @@ public function add_reservation($borr_card, $pin_code, $reservation) {
     $params = array(
       'borrCard' => $borr_card,
       'pinCode' => $pin_code,
-      'reservable' => rawurlencode($reservation['id']),
+      'reservable' => $reservation['id'],
```

Comment:
It's not... it's due to the fact that all alma code is copied for previous solutions and this just got into the mix. Removed.

Nice catch :-)